### DISCUSSION TITLE
Trying to get additionalJobCriteria functioning

### DISCUSSION THREAD

**Discussion by randoloid** [bullhorn/career-portal#524] [Q&A]
we have set up a customText8 field named industry with values of Dental and Not Dental - have attached a screenshot

![bullhorn-screen-capture](https://github.com/bullhorn/career-portal/assets/1241856/81d35ac6-ffd5-4538-9ccf-f9e75facf0b9)
I've updated my app.json file as shown below. 
 "additionalJobCriteria": {
    "field": "customText8",
    "values": [
      "Dental"
    ],
    "sort": "-dateLastPublished"
    
    without those settings everything functions... and from all I've read, I'm configuring this properly.  What am I doing wrong?

**Comment by pbearne** [comment]
Hi Randoloid

I am sure you have been looking at the support  topic https://supportforums.bullhorn.com/viewtopic.php?t=31497
Not seeing it getting fixed in that thread :-)

If you are doing this on a WordPress site, this is a fixed problem in Matador Jobs plugin
We can import the field and 2 pages, one for each option, or filter the whole install via the field

paul@matadorjobs.com
